### PR TITLE
Fix d16-8 Build

### PR DESCRIPTION
**Pull request by jonpryor**
"Environmental changes" broke the d16-8 build, with fixes on master.

CP the fixes so d16-8 can build on CI.

### CODE REVIEW

**Review by radekdoulik — COMMENTED** [review]
LGTM, it has triggered only 2 checks though, which doesn't seem right?

I will add HomeBrew fix to see whether it would trigger more checks.

**Comment by radekdoulik** [comment]
Looks like the Homebrew fix might not be needed in d16-8 as it is not running `xaprep` near the fix location.

**Review by dellis1972 — APPROVED** [review]

File: build-tools/automation/azure-pipelines.yaml [hunk 1]
```diff
@@ -1126,3 +1126,28 @@ stages:
   - template: yaml-templates\run-timezoneinfo-tests.yaml
     parameters:
       node_id: 3
+
+  # Check - "Xamarin.Android (Finalize Installers Queue Vsix Signing)"
+  - job: queue_vsix_signing
```
Comment (pjcollins) [inline build-tools/automation/azure-pipelines.yaml:1131]: We've moved things around a bit in master, but the Jenkins sign job does exist on d16-8:
https://github.com/xamarin/xamarin-android/blob/d16-8/build-tools/automation/azure-pipelines.yaml#L435

We should replace that job with the changes below.

Comment (mjbond-msft) [reply build-tools/automation/azure-pipelines.yaml]: I made the appropriate fixups per https://github.com/xamarin/xamarin-android/pull/5392/commits/cbca46c14eea0ed6ae5814e14db6fd8848046f6f

**Review by pjcollins — APPROVED** [review]

**Review by dellis1972 — APPROVED** [review]

**Pull request merged**